capture.py

```diff
@@ -126,9 +136,10 @@ def capture(camera=None,
 
         with contextlib.nested(
              _maintain_camera(panel, camera),
-             _applied_viewport_options(viewport_options, panel),
```

Comment:
Apply a preset.. you mean applying the settings of a capture-preset onto an actual viewport? Basically the inverse of `parse_view()`?